lib/cib/cib_ops.c

```diff
@@ -579,15 +579,18 @@ update_cib_object(xmlNode * parent, xmlNode * update)
             if (replace[lpc] == ',' || replace[lpc] == 0) {
                 if (last != lpc) {
                     char *replace_item = strndup(replace + last, lpc - last);
-                    xmlNode *remove = find_xml_node(target, replace_item,
-                                                    FALSE);
 
-                    if (remove != NULL) {
-                        crm_trace("Replacing node <%s> in <%s>",
-                                  replace_item, target->name);
-                        free_xml(remove);
+                    if (replace_item != NULL) {
```

Comment:
why not just return errno on NULL?

Reply:
Yeah I can do that. I was thinking preserve existing behavior; this whole block is deprecated anyway.